vendor/engines/secure_rooms/app/controllers/secure_rooms/secure_room_occupancies_controller.rb

```diff
@@ -0,0 +1,13 @@
+module SecureRooms
+
+  class SecureRoomOccupanciesController < ApplicationController
```

Comment:
I don't remember if I had a real reason, but now I prefer it the other way, will change